tg/models/__init__.py

```diff
@@ -173,6 +173,7 @@ def __init__(self, tg):
         self.tg = tg
         self.msgs = defaultdict(list)  # Dict[int, list]
```

Comment:
Let's keep dict of dict where key will be msg_id?

Reply:
Currently `self.msgs` keys are chat ids and value it is list of in order messages. So, it is `Dict[int, List[Dict]]`
We need to have key as a chat_id so we could easily lookup chat messages.
So, we cant use message_id as a key.
Or are you proposing this? `Dict[chat_id, Dict[message_id, Dict]]`?  With this we will use message ordering